source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "set_spec_constant_default_value_pass.h"
 
+#include <algorithm>
```

Comment:
Right.  I think I've been misreading that guideline.   stdio.h would be the C header and cstdio is the corresponding C++ header.  Ooops!
Best not to fight with clang-format!